components/global/Header/_subsubnavigation.scss

```diff
@@ -8,15 +8,23 @@ $_button-bg-color: functions.sass-palette(neutral02);
   height: 100%;
   max-height: 0;
   margin-left: functions.header(nav-link-padding-lateral) / 2;
-  overflow: auto;
+  overflow: hidden;
   color: $_color;
   visibility: hidden;
   transition: max-height 0.25s ease-out, visibility 0s 0.25s;
 
+  @keyframes hide-scroll {
+    from, to {
+      overflow: hidden;
+    }
+  }
+
```

Comment:
super interesting `overflow:hidden` flashy scroll bar hack.  Long and short of it is, `keyframes` also take the most precedence in stylesheet, and so can be effectively used as an override by making the animation duration the same as the `max-height` transition duration.